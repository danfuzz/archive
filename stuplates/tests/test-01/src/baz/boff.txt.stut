I am a ${"processed"} boff.
${zorch}
