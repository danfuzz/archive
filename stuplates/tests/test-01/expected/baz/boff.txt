I am a processed boff.
ZORCH-baz
